packages/flex-plugin-webpack/src/webpack/webpack.config.ts

```diff
@@ -377,11 +384,10 @@ export const _getOptimization = (environment: Environment): Optimization => {
       new TerserPlugin({
         terserOptions: {
           parse: {
-            ecma: 8,
+            ecma: 5,
```

Comment:
Can you please add a comment here why this number is chosen?

Reply:
previously in wp4 the type of ecma was numeric, so it allowed ecma 8
now these are the available ones to choose from
5 | 2015 | 2016 | 2017 | 2018 | 2019 | 2020

Reply:
modified to 2017